doc/source/getting_started.rst

```diff
@@ -0,0 +1,359 @@
+    The axis name must not contain whitespaces and special characters.
+
+.. ipython:: python
+
+    arr2 = ndrange([Axis(["label0", "label1"], "a"), Axis(["label1", "label2"], "b")])
+    arr2
+
+    arr2["label0", x.b["label1"]]
+
+You can also define slices (defined by 'start:stop' or 'start:stop:step').
+A slice will select all labels between `start` and `stop` (stop included).
+All arguments of a slice are optional.
+When not given, start is the first label of an axis, stop the last one.
+
+.. ipython:: python
+
+    # "b1":"b3" is a shortcut for ["b1", "b2", "b3"]
+    # equivalent to: arr["a0,a2", "b1:b3"]
+    arr[["a0", "a2"], "b1":"b3"]
+
+    # :"a2" will select all labels between the first one and "a2"
+    # "b1": will select all labels between "b1" and the last one
+    # equivalent to: arr[":a2", "b1:"]
+    arr[:"a2", "b1":]
+
+Aggregation
+-----------
+
+The LArray library includes many aggregations methods.
+For example, to calculate the sum along an axis, write:
+
+.. ipython:: python
+
+    arr_3D
+
+    # equivalent to: arr_3D.sum("a")
+    arr_3D.sum(x.a)
+
+To aggregate along all axes except one, you simply have to append `_by`
+to the aggregation method you want to use:
+
+.. ipython:: python
+
+    # equivalent to: arr_3D.sum_by("a")
+    arr_3D.sum_by(x.a)
+
+See :ref:`here <la_agg>` to get the list of all available aggregation methods.
+
+Groups
+------
+
+A :ref:`Group <api-group>` represents a subset of labels or positions of an axis:
+
+.. ipython:: python
+
+    arr
+
+    even = arr.a["a0", "a2"]
+    even
+
+    odd = arr.a["a1", "a3"]
+    odd
+
+They can be used in selections:
+
+.. ipython:: python
+
+    arr[even]
+
+    arr[odd]
+
+or aggregations:
+
+.. ipython:: python
+
+    arr.sum((even, odd))
+
+In the case of aggregations, it is often useful to attach them a name
+using the ``>>`` operator:
+
+.. ipython:: python
+
+    # equivalent to: arr.sum("a0,a2 >> even; a1,a3 >> odd")
+    arr.sum((even >> "even", odd >> "odd"))
+
+Group arrays in Session
+-----------------------
+
+Arrays may be grouped in :ref:`Session <api-session>` objects.
+A session is an ordered dict-like container of LArray objects with special I/O methods.
+To create a session, you need to pass a list of pairs (array_name, array):
+
+.. ipython:: python
+
+    arr0 = ndtest((3, 3))
+    arr1 = ndtest((2, 4))
+    arr2 = ndtest((4, 2))
+
+    arrays = [("arr0", arr0), ("arr1", arr1), ("arr2", arr2)]
+    ses = Session(arrays)
+
+    # displays names of arrays contained in the session
+    ses.names
+    # get an array
+    ses["arr0"]
+    # add/modify an array
+    ses["arr3"] = ndtest((2, 2, 2))
+
+.. warning::
+
+    You can also pass a dictionary to the Session's constructor but since elements of a dict object are
+    not ordered by default, you may lose the order. Another way is to declare an empty Session and to add
+    arrays one by one with the syntax: `session['key'] = array`.
```

Comment:
I would remove the whole sentence starting with "Another way..." and replace it by: "If you are using python 3.6 or later, using keyword arguments is a nice alternative which keeps ordering. For example, the session above can be defined using: `ses = Session(arr0=arr0, arr1=arr1, arr2=arr2)`"